CONTRIBUTING.md

```diff
@@ -70,3 +41,15 @@ $ ts-node wormhole-connect/scripts/ofac/getSdnList.ts
 ```
 
 Copy the outputted list and paste it into `src/consts/wallet.ts` as the value of `SANCTIONED_WALLETS`.
+
+## Publishing a pre release package to NPM
+
+1. Make a GitHub pre-release
+2. Tag it `development@0.1.3-beta.0`
+3. Trigger the 'publish' GitHub action, which will publish to NPM
```

Comment:
nitpick: the manual trigger will prompt for the tag to release, so you'll have to specify it

Reply:
added!